dockerstuff/dokument/gitdiff

```diff
@@ -0,0 +1,191 @@
+diff --git a/pom.xml b/pom.xml
```

Comment:
Trengs denne? :P

Reply:
Ja desverre. `token-validation-test-support`-pakken ødelegger for token exchange bare ved å være installert. Noe av det andre her kunne man klart med en egen profil og en egen launcher-klasse, men siden jeg allerede må endre på pom.xml ble det enklest å bare ha de små nødvendige endringene i en gitdiff